### PR TITLE
[OldDot Rules Migration] Individual expense rules

### PR DESCRIPTION
**Pull request by WojtekBoman**
### Details

This PR adds a new section to the Rules Page: `Individual expense rules`. This section allows the user to:
- update the receipt required amount
- update the max expense amount
- update the max expense age
-  enable or disable the billable mode
- enable or disable eReceipts

Implementation details on how the actions listed above work can be found [here](https://docs.google.com/document/d/1oLr14YhL6Y0N5g4tbozdIIrFbybBlsRA0H9I8Wm--w8/edit#heading=h.dcqpif7gm3ei)

### Fixed Issues

$ https://github.com/Expensify/App/issues/47013
PROPOSAL:

### Tests

- [ ] Verify that no errors appear in the JS console

Precondition: Make sure you have access to the workspaceRules beta. If you run the app locally you can modify the `canUseWorkspaceRules` or `canUseAllBetas` function in `Permissions.ts` to always return `true`.

1. Open the settings of any workspace.
2. Enable rules if they are disabled for the selected workspace. This feature can be enabled on the More Features page. 
3. Check that the buttons in the first section work correctly in offline and online mode (check that values ​​are updated correctly, that the correct screens are opened, etc.)

### QA Steps

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/user-attachments/assets/9a531778-5815-49f6-a537-6922f48742b3

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/user-attachments/assets/28159c12-f373-44fe-b163-e328718313c7

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/user-attachments/assets/bc265547-0118-481c-aee1-2a96288949f9

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/user-attachments/assets/c8f351b5-fbe0-44a5-9452-7187284e03d4

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/ec688e85-eac2-414b-a07e-325c09a642aa

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/user-attachments/assets/474d130b-04b2-4423-bc60-eee5cae416e9

</details>

### CODE REVIEW

**Comment by WojtekBoman** [comment]
@marcaaron 

I have one question. When we upgrade the policy, default max expense amounts are set:

```
function upgradeToCorporate(policyID: string, featureName: string) {
    const policy = getPolicy(policyID);
    const optimisticData: OnyxUpdate[] = [
        {
            onyxMethod: Onyx.METHOD.MERGE,
            key: `policy_${policyID}`,
            value: {
                isPendingUpgrade: true,
                type: CONST.POLICY.TYPE.CORPORATE,
                // HERE DEFAULTS ARE SET
                maxExpenseAge: CONST.POLICY.DEFAULT_MAX_EXPENSE_AGE,
                maxExpenseAmount: CONST.POLICY.DEFAULT_MAX_EXPENSE_AMOUNT,
                maxExpenseAmountNoReceipt: CONST.POLICY.DEFAULT_MAX_AMOUNT_NO_RECEIPT,
                glCodes: true,
                ...(PolicyUtils.isInstantSubmitEnabled(policy) && {
                    autoReporting: true,
                    autoReportingFrequency: CONST.POLICY.AUTO_REPORTING_FREQUENCIES.IMMEDIATE,
                }),
                harvesting: {
                    enabled: false,
                },
            },
        },
    ];
    
    // CONST.ts
    
      DEFAULT_MAX_EXPENSE_AGE: 90,
      DEFAULT_MAX_EXPENSE_AMOUNT: 200000,
      DEFAULT_MAX_AMOUNT_NO_RECEIPT: 2500,    
```

Is this code still valid or should we set all these values to 10000000000?

File: src/languages/es.ts [hunk 1]
```diff
@@ -3648,6 +3649,24 @@ export default {
             individualExpenseRules: {
                 title: 'Gastos',
                 subtitle: 'Establece controles y valores predeterminados para gastos individuales. También puedes crear reglas para',
+                receiptRequiredAmount: 'Monto requerido para recibo',
+                receiptRequiredAmountDescription: 'Requiere recibos cuando el gasto excede este monto, a menos que una regla de categoría lo anule.',
```
Comment (pecanoro) [inline src/languages/es.ts:3653]:
```suggestion
                receiptRequiredAmount: 'Cantidad requerida para los recibos',
                receiptRequiredAmountDescription: 'Exige recibos cuando los gastos superen este importe, a menos que lo anule una regla de categoría.',
```

File: src/languages/es.ts [hunk 2]
```diff
@@ -3648,6 +3649,24 @@ export default {
...
+                maxExpenseAmount: 'Monto máximo de gasto',
+                maxExpenseAmountDescription: 'Marca el gasto que excede este monto, a menos que una regla de categoría lo anule.',
```
Comment (pecanoro) [inline src/languages/es.ts:3655]:
```suggestion
                maxExpenseAmount: 'Importe máximo del gasto',
                maxExpenseAmountDescription: 'Marca los gastos que superen este importe, a menos que una regla de categoría lo anule.',
```

File: src/languages/es.ts [hunk 3]
```diff
@@ -3648,6 +3649,24 @@ export default {
...
+                maxAge: 'Edad máxima',
+                maxExpenseAge: 'Antigüedad máxima de gasto',
```
Comment (pecanoro) [inline src/languages/es.ts:3657]:
```suggestion
                maxAge: 'Antigüedad máxima',
                maxExpenseAge: 'Antigüedad máxima de los gastos',
```

File: src/languages/es.ts [hunk 4]
```diff
@@ -3648,6 +3649,24 @@ export default {
...
+                maxExpenseAgeDescription: 'Marca los gastos con más de un número específico de días.',
```
Comment (pecanoro) [inline src/languages/es.ts:3658]:
```suggestion
                maxExpenseAgeDescription: 'Marca los gastos de más de un número determinado de días.',
```

File: src/languages/es.ts [hunk 5]
```diff
@@ -3648,6 +3649,24 @@ export default {
...
+                maxExpenseAgeDays: (age: number) => `${age} ${Str.pluralize('día', 'días', age)}`,
+                billableDefault: 'Valor predeterminado facturable',
+                billableDefaultDescription:
+                    'Elige si los gastos en efectivo y con tarjeta de crédito deben ser facturables por defecto. Los gastos facturables se habilitan o deshabilitan en etiquetas',
```
Comment (pecanoro) [inline src/languages/es.ts:3662]:
```suggestion
                    'Elige si los gastos en efectivo y con tarjeta de crédito deben ser facturables por defecto. Los gastos facturables se activan o desactivan en',
```

File: src/languages/es.ts [hunk 6]
```diff
@@ -3648,6 +3649,24 @@ export default {
...
+                billable: 'Facturable',
+                billableDescription: 'Los gastos se vuelven a facturar a los clientes con mayor frecuencia',
```
Comment (pecanoro) [inline src/languages/es.ts:3664]:
```suggestion
                billableDescription: 'Los gastos se vuelven a facturar a los clientes en la mayoría de los casos',
```

File: src/languages/es.ts [hunk 7]
```diff
@@ -3648,6 +3649,24 @@ export default {
...
+                nonBillable: 'No facturable',
+                nonBillableDescription: 'Los gastos ocasionalmente se vuelven a facturar a los clientes',
```
Comment (pecanoro) [inline src/languages/es.ts:3666]:
```suggestion
                nonBillableDescription: 'Los gastos se vuelven a facturar a los clientes en ocasiones',
```

File: src/languages/es.ts [hunk 8]
```diff
@@ -3648,6 +3649,24 @@ export default {
...
+                eReceipts: 'Recibos electrónicos',
+                eReceiptsHint: 'El Recibos electrónicos se crean automáticamente para',
+                eReceiptsHintLink: 'la mayoría de las transacciones con crédito en USD',
```
Comment (pecanoro) [inline src/languages/es.ts:3669]:
```suggestion
                eReceiptsHint: 'Los recibos electrónicos se crean automáticamente',
                eReceiptsHintLink: 'para la mayoría de las transacciones en USD',
```

**Review by pecanoro — CHANGES_REQUESTED** [review]

**Comment by shawnborton** [comment]
Screenshots look pretty good to me!

**Comment by marcaaron** [comment]
>Is this code still valid or should we set all these values to 10000000000?

@WojtekBoman What you have there appears to be correct. When the API is called to upgrade to corporate those are the default values we set.

File: src/libs/API/types.ts [hunk 9]
```diff
@@ -201,6 +201,11 @@ const WRITE_COMMANDS = {
     ENABLE_POLICY_EXPENSIFY_CARDS: 'EnablePolicyExpensifyCards',
     ENABLE_POLICY_INVOICING: 'EnablePolicyInvoicing',
     SET_POLICY_RULES_ENABLED: 'SetPolicyRulesEnabled',
+    SET_POLICY_EXPENSE_MAX_AMOUNT_NO_RECEIPT: 'SetPolicyExpenseMaxAmountNoReceipt',
+    SET_POLICY_EXPENSE_MAX_AMOUNT: 'SetPolicyExpenseMaxAmount',
+    SET_POLICY_EXPENSE_MAX_AGE: ' SetPolicyExpenseMaxAge',
+    SET_POLICY_BILLABLE_MODE: ' SetPolicyBillableMode',
+    SET_WORKSPACE_ERECEIPTS_ENABLED: 'SetWorkspaceEReceiptsEnabled',
```
Comment (marcaaron) [inline src/libs/API/types.ts:208]: Note to myself that I will create some issue to fix this as polish.

**Comment by marcaaron** [comment]
Looks great so far!

**Comment by brunovjk** [comment]
I don't have beta access right now, so I set `canUseWorkspaceRules` to `true`. However, the fields in the _Individual expense rules_ section show default values instead of being blank, as shown in the PR screenshot:

<details>
<summary>video</summary>



https://github.com/user-attachments/assets/3ff82230-2883-43e8-8b68-57b5297d7363

</details>

Screenshot 2024-08-24 at 14 59 51

The fields are editable, and everything else works well. Could this behavior be related to not having beta access? Should we be concerned about this? Thanks.

**Comment by WojtekBoman** [comment]
> I don't have beta access right now, so I set `canUseWorkspaceRules` to `true`. However, the fields in the _Individual expense rules_ section show default values instead of being blank, as shown in the PR screenshot:
> 
> video
> Screenshot 2024-08-24 at 14 59 51
> 
> The fields are editable, and everything else works well. Could this behavior be related to not having beta access? Should we be concerned about this? Thanks.
> 
> cc: @WojtekBoman @marcaaron

I mentioned it [here](https://github.com/Expensify/App/pull/47409#issuecomment-2307050949). When we upgrade our policy, the default values for rules are set

**Comment by brunovjk** [comment]
> @WojtekBoman What you have there appears to be correct. When the API is called to upgrade to corporate those are the default values we set.

Sorry, I missed that. I'm testing on all platforms and will fill out the checklist.

**Comment by brunovjk** [comment]
Excuse @WojtekBoman, what should happen when we save a _'max expense age'_ equal to the [maximum limit](https://github.com/Expensify/App/blob/f7c9fa8f7698b17af86e566abe59b46a33a19bc1/src/CONST.ts#L2498)? Thanks.


https://github.com/user-attachments/assets/5c9f4fd2-7019-4a9d-aedc-6953e01b5d09

**Comment by WojtekBoman** [comment]
> Excuse @WojtekBoman, what should happen when we save a _'max expense age'_ equal to the [maximum limit](https://github.com/Expensify/App/blob/f7c9fa8f7698b17af86e566abe59b46a33a19bc1/src/CONST.ts#L2498)? Thanks.
> 
>  Screen.Recording.2024-08-26.at.10.29.55.mov

I asked about it [here](https://github.com/Expensify/App/issues/47013#issuecomment-2303976268) @marcaaron Could you take a look at it?

**Comment by brunovjk** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



https://github.com/user-attachments/assets/a03c2bb9-306b-4fdd-abcf-9d3b3ecd9eb8


</details>

<details>
<summary>iOS: mWeb Safari</summary>



https://github.com/user-attachments/assets/48453dce-a8ef-4dc5-b056-e35499305659


</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



https://github.com/user-attachments/assets/9ebd25c3-17e5-43b4-beda-97667ee00aad


</details>

<details>
<summary>MacOS: Desktop</summary>



https://github.com/user-attachments/assets/d3749738-9ff2-4c5f-8957-8d2cc8882a19


</details>

**Review by brunovjk — APPROVED** [review]
Miding those comments: [#comment_1](https://github.com/Expensify/App/pull/47409#issuecomment-2310251088) and [#comment_2](https://github.com/Expensify/App/pull/47409#issuecomment-2310413404) everything else LGTM.

**Review by pecanoro — APPROVED** [review]

**Review by marcaaron — APPROVED** [review]
LGTM

**Pull request merged**